### PR TITLE
Make app version dynamic

### PR DESCRIPTION
**Pull request by jareware**

### CODE REVIEW

**Comment by cutepig** [comment]
Does this affect any instructions in the _Release process_ section in README.md? https://github.com/futurice/symptomradar/blob/master/README.md#release-process

**Comment by jareware** [comment]
Yeeeesss it does! Great point. I'll fix that.

File: scripts/app-version-get [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+set -euo pipefail # exit on error; treat unset variables as errors; exit on errors in piped commands
+git describe --exact-match --tags 2>/dev/null || (echo -n "dev-" && git rev-parse --short HEAD) # e.g. "v2.2" or "dev-a821087"
```
Comment (cutepig) [inline scripts/app-version-get:4]: How would you end up with `v2.2`? From git tags?

File: scripts/deploy-frontend [hunk 2]
```diff
@@ -11,12 +11,18 @@ fi
 
 set -euo pipefail # exit on error; treat unset variables as errors; exit on errors in piped commands
 SELF_DIR="$(perl -e 'use File::Basename; use Cwd "abs_path"; print dirname(abs_path(@ARGV[0]));' -- "$0")" # figure out the absolute path to the current script, regardless of pwd (perl is more cross-platform than realpath; https://stackoverflow.com/a/30795461)
+ROOT_DIR="$SELF_DIR/.."
```
Comment (cutepig) [inline scripts/deploy-frontend:14]: Is there a reason why we don't rely on plain `pwd`? I typically orchestrate everything to be run on the root project directory, but is there a reason why this wouldn't be the case? Do some hosting platforms do things differently?

File: src/index-backend.ts [hunk 3]
```diff
@@ -1,16 +1,24 @@
...
+  prepareResponseForStorage,
+  storeDataDumpsToS3,
+  storeResponse,
+  updateOpenDataIndex,
+} from './backend/main';
 import { assertIs, FrontendResponseModel, FrontendResponseModelT } from './common/model';
 
 const dynamoDb = createDynamoDbClient(process.env.ABUSE_DETECTION_TABLE || '');
 
+console.log(`Backend ${APP_VERSION} started`);
```
Comment (cutepig) [inline src/index-backend.ts:15]: Very nice :+1:

File: scripts/app-version-get (hunk 1 above)
Comment (jareware) [reply scripts/app-version-get:4]: Yes! The GitHub release process that we use creates them automatically.

File: scripts/deploy-frontend (hunk 2 above)
Comment (jareware) [reply scripts/deploy-frontend:14]: It's mostly for safety. Having a build script run with wrong assumptions can make a mess. So I prefer to either a) enforce the PWD or b) make it irrelevant.

**Pull request merged**